app/views/forms/show.html.slim

```diff
@@ -0,0 +1,46 @@
+.container
+  .section
+    h1 = @form.title
+
+    = form_with(id: :form, model: @form) do |form|
+      = form.hidden_field :form_type, value: @form.form_type
+
+      - @form.questions.each do |question|
+        - if question[:options].is_a?(Array)
+          = field_set_tag nil, "cf-questions" => question[:question] do
+            - question[:options].each do |option|
+              = form.radio_button question[:key], option, required: question[:required], "cf-label" => option
+
+        - else
+          = form.text_field question[:key], required: question[:required], "cf-questions" => question[:question]
+
+    div[id="cf-context" role="cf-context" cf-context=true]
+
+script[type="text/javascript" src="https://cdn.jsdelivr.net/gh/space10-community/conversational-form@1.0.1/dist/conversational-form.min.js" crossorigin=true]
+
+css:
+  /*
+    TODO: Move this into asset pipeline. For some reason putting styles in
+      base.scss does not get picked up.
+  */
+  html {
+    scroll-behavior: smooth;
+  }
+
+  #cf-context {
+    width: 100%;
+    height: 500px;
+  }
+
+  #form {
+    position: absolute;
+    height: 0px;
+    visibility: hidden;
+  }
+
+javascript:
+  var conversationalForm = window.cf.ConversationalForm.startTheConversation({
+    formEl: document.getElementById("form"),
+    context: document.getElementById("cf-context"),
+    theme: 'dark'
+  })
```

Comment:
perhaps we can set `robotImage` to be RubySG logo instead?
one downside is that I personally tried and can't get `theme` and `robotImage` to work concurrently tho

Reply:
yea. lets go without the image for now. I've changed the theme to red because I think it suits RubySG theme better.
<img width="1182" alt="Screenshot 2021-09-14 at 11 06 45 PM" src="https://user-images.githubusercontent.com/10722197/133283441-2405f1be-e719-4b00-9da7-d8d83351e0e9.png">